src/components/pages/DataViz/bridge_card.js

```diff
@@ -42,128 +40,109 @@ const BridgeCard = () => {
-                      {detailsData.province}
-                    </p>
-                    <p>
-                      <span className="bottomInfoTags">District:</span>{' '}
-                      {detailsData.district}
-                    </p>
-                    <p>
-                      <span className="bottomInfoTags">Bridge Type:</span>{' '}
-                      {detailsData.type}
-                    </p>
-                    <p>
-                      <span className="bottomInfoTags">Project Sub Stage:</span>{' '}
-                      {detailsData.sub_stage}
-                    </p>
-                    <p>
-                      <span className="bottomInfoTags">Project Stage:</span>{' '}
-                      {detailsData.stage}
-                    </p>
-                  </div>
-                  <div className="bottom_info_cols">
-                    <p>
-                      <span className="bottomInfoTags">
-                        Estimate of People Served:
-                      </span>{' '}
-                      {detailsData.Individuals_directly_served}
-                    </p>
-                    <p>
-                      <span className="bottomInfoTags">
-                        AVG Households Served:
-                      </span>{' '}
-                      {detailsData.inc_income}
-                    </p>
-                    <p>
-                      <span className="bottomInfoTags">
-                        Economic Impact (RWF):
-                      </span>{' '}
-                      {detailsData.inc_income_rwf}
-                    </p>
-                    <p>
-                      <span className="bottomInfoTags">
-                        Economic Impact (USD):
-                      </span>{' '}
-                      {detailsData.inc_income_usd}
-                    </p>
-                  </div>
-                </div>
-              </Typography>
-            </CardContent>
-          </Collapse>
-        </Card>
-      </div>
-    </Draggable>
+        <CardContent>
+          <Typography paragraph>
+            <div className="bottom_info">
+              <div className="bottom_info_cols">
+                <p>
+                  <span className="bottomInfoTags">Province:</span>{' '}
+                  {detailsData.province}
+                </p>
+                <p>
+                  <span className="bottomInfoTags">District:</span>{' '}
+                  {detailsData.district}
+                </p>
+                <p>
+                  <span className="bottomInfoTags">Project Sub Stage:</span>{' '}
+                  {detailsData.sub_stage}
+                </p>
+                <p>
+                  <span className="bottomInfoTags">Project Stage:</span>{' '}
+                  {detailsData.project_stage}
+                </p>
+              </div>
+              <div className="bottom_info_cols">
+                {/* currently unsure if following info could be implemented */}
```

Comment:
Maybe consider adding this to Trello so we can keep the code clean and not forget about this in the future